### ISSUE TITLE
Quick Fix hover not working

### ISSUE THREAD

**Issue by archer3cl** [microsoft/vscode-eslint#706]
After updating to VS Code 1.36.0 the option for quick fix when hovering does not apply the related action.
![NOACTION](https://user-images.githubusercontent.com/4583932/60730600-270b0100-9f0b-11e9-84ef-5a3a05379c42.gif)

Using the quick fix on the bulb action does.
![ACTION](https://user-images.githubusercontent.com/4583932/60730671-4a35b080-9f0b-11e9-8fe9-2b6b7b6f3700.gif)

VS Code Info
Version: 1.36.0 (user setup)
Commit: 0f3794b38477eea13fb47fbe15a42798e6129338
Date: 2019-07-03T13:25:46.372Z
Electron: 4.2.5
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Comment by MatiasOlivera** [comment]
I'm having the same problem after updating VSCode. I'm going to use the bulb action meanwhile.

```
Version: 1.36.0 (user setup)
Commit: 0f3794b38477eea13fb47fbe15a42798e6129338
Date: 2019-07-03T13:25:46.372Z
Electron: 4.2.5
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.17763
```

**Comment by zjb19970804** [comment]
Yes, after updating the latest version of vscode IDE today, eslink has changed. It used to be normal, but I can use Ctrl +. to fix code errors.

**Comment by dbaeumer** [comment]
@mjbvz same as with the Java plugin. Has something changed here in VS Code ?

**Comment by dbaeumer** [comment]
I move the issue to VS Code since the quick fix action computation is the same in both pathes.
